test/installStocktrader.sh

```diff
@@ -0,0 +1,21 @@
+isKAppNav=$1
+
+echo "kubectl create namespace stocktrader"
+kubectl create namespace stocktrader
+
+if [ x$isKAppNav == 'x' ]; then
```

Comment:
Let's talk about how we can avoid referring to tmpbuild and having a isKAppNav param.